debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/commands/ShrCCommand.java

```diff
@@ -1202,9 +1202,14 @@ ShrcConfig dbgShrcReadConfig(J9SharedClassConfigPointer sharedClassConfig, Print
 					/*  _ccHead -------------> ccNext ---------> ccNext --------> ........---------> ccTail
 					 * (top layer)         (middle layer)     (middle layer)      ........         (layer 0)
 					 */
-					SH_CompositeCacheImplPointer compositeCacheImpl = cacheMap._ccTail();
-					for (int tmplayer = 0; tmplayer < layer; tmplayer++) {
-						compositeCacheImpl = compositeCacheImpl._previous();
+					SH_CompositeCacheImplPointer compositeCacheImpl = null;
```

Comment:
Please remove the unnecessary initialization to `null` (the java compiler will issue an error if it's not guaranteed to have a value before it's used).

Reply:
Fixed.